From: Arjan van de Ven <arjan@linux.intel.com>

Reduce system wakeups by not waking up PHP once per second


--- php-5.6.17/sapi/fpm/fpm/fpm_events.c~	2016-01-06 10:14:47.000000000 -0500
+++ php-5.6.17/sapi/fpm/fpm/fpm_events.c	2016-03-05 10:38:44.907110624 -0500
@@ -402,7 +402,7 @@
 
 		/* 1s timeout if none has been set */
 		if (!timerisset(&ms) || timercmp(&ms, &now, <) || timercmp(&ms, &now, ==)) {
-			timeout = 1000;
+			timeout = 30000;
 		} else {
 			timersub(&ms, &now, &tmp);
 			timeout = (tmp.tv_sec * 1000) + (tmp.tv_usec / 1000) + 1;
--- php-5.6.17/sapi/fpm/fpm/fpm_process_ctl.c~	2016-01-06 10:14:47.000000000 -0500
+++ php-5.6.17/sapi/fpm/fpm/fpm_process_ctl.c	2016-03-12 18:15:18.726600269 -0500
@@ -191,7 +191,7 @@
 		} else {
 			sig = SIGKILL;
 		}
-		timeout = 1;
+		timeout = 10;
 	}
 
 	fpm_pctl_kill_all(sig);
